ENIGMAsystem/SHELL/Graphics_Systems/OpenGL1/GLscreen.cpp

```diff
@@ -305,6 +307,7 @@ void screen_redraw()
     // glReadPixels(0, 0, width, height, GL_BGRA, GL_UNSIGNED_BYTE, pixels);
   }
 
+  draw_sprite(cursor_sprite, 0, mouse_x, mouse_y);
```

Comment:
Did you test this? As far as I can tell, `draw_sprite` has no special handling for the `-1` case. This should cause all kinds of problems.

Reply:
Yes and that's why I have "SEGMENTATION FAULT" in bold letters in the OP above. GameMaker defaults it to -1, I don't know what you want me to do, check if it's > 0 before calling draw_sprite?